test/unit/app/presenters/publishing_api/organisations_index_presenter_test.rb

```diff
@@ -0,0 +1,146 @@
+require "test_helper"
+
+class PublishingApi::OrganisationsPresenterTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  test "presents a valid content item" do
+    organisation_one = create(:organisation, organisation_type_key: :executive_office, content_id: "e08eb3d9-ade7-4a5a-9b69-35008220b77f")
+    organisation_two = create(:organisation, organisation_type_key: :ministerial_department, content_id: "0ad3f6c2-5492-4d37-86cf-27afb8d85b1c")
+    organisation_three = create(:organisation, organisation_type_key: :non_ministerial_department, content_id: "01ba16a6-2fd6-4621-9296-8afbd3114c1b")
+    organisation_four = create(:organisation, organisation_type_key: :public_corporation, content_id: "735b5c03-7937-4c68-920b-d087795fccd9")
```

Comment:
Do you need to specify content IDs here?  I think the factories should generate one automatically.

Reply:
This is related to the issue I discussed with you with the generated titles changing, now I've solved that I can dispense with this as well. Cheers for pointing it out.